changelog/5249.breaking.1.rst

```diff
@@ -0,0 +1 @@
+Deprecated `sunpy.roi.chaincode` and created a replacement at `sunpy.net.helio.chaincode`.
```

Comment:
Is the replacement version not a drop-in alternative?

Reply:
Nope.

Reply:
What changed? We should *at least* document the changes if we have changed stuff.

Reply:
I changed the function names.